<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEW_Propeties</class>
 <widget class="QWidget" name="GEW_Propeties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="ToolBarLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	background-color:transparent;
    border-bottom-left-radius: 3px;
    border-bottom-right-radius: 3px;
    border: 1px solid #acacac;
    border-top: none;
    margin-top: 18px;
}
 
QGroupBox::title
{
	subcontrol-origin: margin;	
    background:url(:/assets/subWindowBackground);
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #0b477b;
    border-bottom: none;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #005192, stop: 1 #002f60);
    padding: 2 5000 2 2; /* some large value, as QGrouBox actual width is not known */
    color: white; 
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>10</number>
     </property>
     <widget class="QWidget" name="EnvironmentProbPropertyPage">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_EnvironmentProbeProperty" name="EnvironmentProbProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="LightPropertyPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_LightProperty" name="LightProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FlarePropertyPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_FlareProperty" name="FlareProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ModelPropertyPage">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_ModelProperty" name="ModelProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PostprocessPropertyPage">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_PostprocessProperty" name="PostprocessProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DecalPropertyPage">
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_DecalProperty" name="DecalProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TerrainPropertyPage">
      <layout class="QGridLayout" name="gridLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_TerrainProperty" name="TerrainProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ScreenshotPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_ScreenshotProperty" name="ScreenshotProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DefaultCameraPage">
      <layout class="QGridLayout" name="gridLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_DefaultCameraProperty" name="DefaultCameraProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EntityPropertyPage">
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GEW_EntityPtoperty" name="EntityProperty" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EmptyPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="NoItem">
         <property name="text">
          <string>No Item Selected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="PostProcess">
   <property name="icon">
    <iconset resource="GEditor.qrc">
     <normaloff>:/Resources/MetroStyle/widescreen_tv.png</normaloff>:/Resources/MetroStyle/widescreen_tv.png</iconset>
   </property>
   <property name="text">
    <string>PostProcess</string>
   </property>
   <property name="toolTip">
    <string>Show PostProcess property</string>
   </property>
  </action>
  <action name="Terrain">
   <property name="icon">
    <iconset resource="GEditor.qrc">
     <normaloff>:/Resources/MetroStyle/Height.png</normaloff>:/Resources/MetroStyle/Height.png</iconset>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
   <property name="toolTip">
    <string>Show terrain property</string>
   </property>
  </action>
  <action name="Screenshot">
   <property name="icon">
    <iconset resource="GEditor.qrc">
     <normaloff>:/Resources/MetroStyle/compact_camera.png</normaloff>:/Resources/MetroStyle/compact_camera.png</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Show screenshot property</string>
   </property>
  </action>
  <action name="DefaultCamera">
   <property name="icon">
    <iconset resource="GEditor.qrc">
     <normaloff>:/Resources/MetroStyle/camcoder_pro.png</normaloff>:/Resources/MetroStyle/camcoder_pro.png</iconset>
   </property>
   <property name="text">
    <string>DefaultCamera</string>
   </property>
   <property name="toolTip">
    <string>Show default camera property</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GEW_LightProperty</class>
   <extends>QWidget</extends>
   <header>GEW_LightProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_FlareProperty</class>
   <extends>QWidget</extends>
   <header>GEW_FlareProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_ModelProperty</class>
   <extends>QWidget</extends>
   <header>GEW_ModelProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_PostprocessProperty</class>
   <extends>QWidget</extends>
   <header>GEW_PostprocessProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_DecalProperty</class>
   <extends>QWidget</extends>
   <header>GEW_DecalProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_EnvironmentProbeProperty</class>
   <extends>QWidget</extends>
   <header>GEW_EnvironmentProbeProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_TerrainProperty</class>
   <extends>QWidget</extends>
   <header>GEW_TerrainProperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_ScreenshotProperty</class>
   <extends>QWidget</extends>
   <header>gew_screenshotproperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_DefaultCameraProperty</class>
   <extends>QWidget</extends>
   <header>gew_defaultcameraproperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GEW_EntityPtoperty</class>
   <extends>QWidget</extends>
   <header>GEW_EntityPtoperty.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GEditor.qrc"/>
 </resources>
 <connections/>
</ui>
